components/Ink/src/MDCInkView.m

```diff
@@ -155,13 +154,6 @@ - (void)layoutSubviews {
       inkLayer.bounds = inkBounds;
     }
   }
-
-  // If the superview has a shadowPath make sure ink does not spread outside of the shadowPath.
-  if (self.superview.layer.shadowPath) {
-    self.maskLayer.path = self.superview.layer.shadowPath;
-    self.layer.mask = _maskLayer;
-    self.layer.masksToBounds = YES;
-  }
```

Comment:
Are we confident we don't need this code anymore?

Reply:
This looks like a merge error: https://github.com/material-components/material-components-ios/pull/2545/commits/d43a7cbad539f2307d842e85a5ab5b0659b7cc22

Reply:
(i remove the https:// because github converted the link into a different one)

Reply:
Be sure to resolve this error before merging the PR.

Reply:
Ah I meant the implementor of Ink added this code twice due to a merge mistake 4 months ago, and I just removed the duplicate :)